### DISCUSSION TITLE
when will Iris with PBR support drop?

### DISCUSSION THREAD

**Discussion by Aaryan-Anand** [IrisShaders/Iris#1591] [General]
I know you guys are working hard to pump out updates, but just out of curiosity, when will the Iris 1.3.0 release drop which has PBR support?

**Comment by NotJansel** [comment]
#130 is a issue already with some infos. else you can go on the Discord and get the PBR Testbuilds from #public-test-builds. Also there is a new Installer in the Works with which you can get the latest PBR Beta without needing to be on the Discord.

**Comment by IMS212** [comment]
This has existed for 6 months now.
